### PR TITLE
feat: support aria-labelledby and aria-describedby for dialog in Panel

### PR DESCRIPTION
**Pull request by kapiljaveri**
The DrawerPanel has a dialog role which should support aria-labelledby and aria-describedby attributes.

### CODE REVIEW

**Comment by kapiljaveri** [comment]
@zombieJ if you can take a look when you get the chance :)

**Comment by afc163** [comment]
Conflicting

**Comment by kapiljaveri** [comment]
@afc163, all resolved.

File: src/DrawerPanel.tsx [hunk 1]
```diff
@@ -16,7 +16,14 @@ export interface DrawerPanelEvents {
   onKeyUp?: React.KeyboardEventHandler<HTMLDivElement>;
 }
 
-export interface DrawerPanelProps extends DrawerPanelEvents {
+export type DrawerPanelAccessibility = Pick<
+  React.DialogHTMLAttributes<HTMLDivElement>,
+  'aria-labelledby' | 'aria-describedby'
```
Comment (yoyo837) [inline src/DrawerPanel.tsx]: Why not `aria-*`?

Comment (kapiljaveri) [reply src/DrawerPanel.tsx]: I could add that, wasn't sure if the current restrictions were in place for some reason I wasn't aware of :)

Comment (kapiljaveri) [reply src/DrawerPanel.tsx]: done

**Comment by afc163** [comment]
We should change [this line](https://github.com/react-component/drawer/pull/425/files#diff-dcff384385205726c2fd9edd210424729b6c64c20dc81db32480768780e3c169R314).

```diff
- {...pickAttrs(props, { data: true })}
+ {...pickAttrs(props, { data: true, aria: true })}
```

**Comment by kapiljaveri** [comment]
@afc163, won't that apply it to the container div instead of the Panel having the div with the role=dialog? The goal was to make the the dialog queryable via the attributes.

But I could definitely leverage that instead of my custom function.

**Comment by afc163** [comment]
@kapiljaveri You are right.

**Comment by kapiljaveri** [comment]
Awesome, thanks for the review!

**Pull request merged**